### PR TITLE
Implements UK Payment Initiation support in Java

### PR DESCRIPTION
**Pull request by skansal-plaid**
Implements UK Payment Initiation support in Java.

Tested locally that both payment-initiation and the default flow work successfully.

### CODE REVIEW

File: java/src/main/java/com/plaid/quickstart/resources/PaymentInitiationResource.java [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+  @GET
+  public PaymentGetResponse getPayment() throws IOException {
+    String paymentId = QuickstartApplication.paymentId;
+
+    Response<PaymentGetResponse> paymentGetResponse =
+      plaidClient.service().paymentGet(new PaymentGetRequest(paymentId)).execute();
+    if (!paymentGetResponse.isSuccessful()) {
+      try {
+        ErrorResponse errorResponse = plaidClient.parseError(paymentGetResponse);
+        Thread.sleep(3000);
```
Comment (jacksiegel) [inline java/src/main/java/com/plaid/quickstart/resources/PaymentInitiationResource.java]: Why call `Thread.sleep`?

File: java/src/main/java/com/plaid/quickstart/resources/PaymentInitiationResource.java [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+      } catch(InterruptedException e) {
+        // catch error
```
Comment (jacksiegel) [inline java/src/main/java/com/plaid/quickstart/resources/PaymentInitiationResource.java]: Should we log the error?

File: java/src/main/java/com/plaid/quickstart/resources/PaymentInitiationResource.java (hunk 1 above)
Comment (skansal-plaid) [reply java/src/main/java/com/plaid/quickstart/resources/PaymentInitiationResource.java]: We should not, it was copy-pasta https://github.com/plaid/quickstart/pull/90/commits/bedafdf1aed41d7b28f047ed8d004dc3ad1d3810

File: java/src/main/java/com/plaid/quickstart/resources/PaymentInitiationResource.java (hunk 2 above)
Comment (skansal-plaid) [reply java/src/main/java/com/plaid/quickstart/resources/PaymentInitiationResource.java]: Good call, done

**Pull request merged**